.vscode/launch.json

```diff
@@ -21,5 +21,29 @@
             "remote": true,
             "cwd": "${workspaceRoot}/build"
         },
+        {
+            "name": "Launch garage-push",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceRoot}/build/src/sota_tools/garage-push",
+            "args": ["--ref", "raspberrypi3-otatk", 
+                "-C", "/home/tolik/prj/rasb/build/tmp/deploy/images/raspberrypi3/ostree_repo",  
+                "-n", 
+                "-j", "/home/tolik/prj/hatcred/credentials.zip", 
+                "--loglevel", "0"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}/build",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                }
+            ]
+        },
```

Comment:
I was never particularly fond of this file being part of the repo in the first place, but if it's going to be there, it really shouldn't have absolute paths that won't work for anyone else. I'd prefer if this file were removed and put it .gitignore, but I'd accept it if the paths were relative.

Reply:
yep. make sense remove the file. anyway configuration for aktualizr does not work